src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -154,11 +154,13 @@ const constructPageFetchUrl = ({
 
   if (isLocal) {
     switch (pageType) {
-      case ARTICLE_PAGE:
-        fetchUrl = Url(
-          `/${service}/articles/${id}${variant ? `/${variant}` : ''}`,
-        );
-        break;
+      case ARTICLE_PAGE: {
+        const isCpsId = /([0-9]{5,9}|[a-z0-9\-_]+-[0-9]{5,9})$/.test(id);
+        if (isCpsId) {
+          return Url(`/${id}`);
+        }
+        return Url(`/${service}/articles/${id}${variant ? `/${variant}` : ''}`);
+      }
```

Comment:
```suggestion
      case ARTICLE_PAGE:
        fetchUrl = Url(
          isCpsId(id)
            ? `/${id}`
            : `/${service}/articles/${id}${variant ? `/${variant}` : ''}`,
        );
        break;
```
The [isCpsId](https://github.com/bbc/simorgh/pull/11529/files#diff-b0291bb91a24efe1af51dd8467d6e14fa879769816d63e1fb5f2622819350b90R158
) check is applied [here](https://github.com/bbc/simorgh/pull/11529/files#diff-b0291bb91a24efe1af51dd8467d6e14fa879769816d63e1fb5f2622819350b90R70) as well.
Consider moving the check to a separate function for the purpose of reuse.
See example below.

```
const isCpsId = (path: string) =>
  /([0-9]{5,9}|[a-z0-9\-_]+-[0-9]{5,9})$/.test(path);
```

Reply:
Good spot - have updated!